style.css

```diff
@@ -0,0 +1,33 @@
+* {
+    margin: 0;
+    padding: 0;
+    box-sizing: border-box;
+  }
+  
+  #title {
```

Comment:
Please avoid using id selector instead use class selectors for CSS best practices.